fixtures/vcr_cassettes/elb-named-test.yml

```diff
@@ -12,15 +12,15 @@
           Accept-Encoding: 
             - ""
           User-Agent: 
-            - "aws-sdk-ruby2/2.0.1 ruby/2.1.1 x86_64-darwin13.0"
+            - "aws-sdk-ruby2/2.0.5 ruby/2.0.0 universal.x86_64-darwin14"
           X-Amz-Date: 
-            - "20141003T114706Z"
+            - "20150212T113953Z"
           Host: 
             - elasticloadbalancing.sa-east-1.amazonaws.com
           X-Amz-Content-Sha256: 
             - "236069f72bf74f0c7ddff0a34b0defa8a21d1d6a897e588764e1b2ff6319f94a"
           Authorization: 
-            - "AWS4-HMAC-SHA256 Credential=redacted/20141003/sa-east-1/elasticloadbalancing/aws4_request, SignedHeaders=content-type;host;user-agent;x-amz-content-sha256;x-amz-date, Signature=redacted"
+            - "AWS4-HMAC-SHA256 Credential=redacted/20150212/sa-east-1/elasticloadbalancing/aws4_request, SignedHeaders=content-type;host;user-agent;x-amz-content-sha256;x-amz-date, Signature=4ce9fa2a07cae0d74fbccb44f024ae2e3d95cfe0e7c851b614230c70d5f5aee3"
```

Comment:
No, Not really. The signature is basically a HMAC one way hash of the request. Without the key id and the key it's not useful. If you had the key id you could replay the exact same request (which would fail likely fail because it's being made in the past).